src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/http_action/config/PerformanceAnalyzerConfigAction.java

```diff
@@ -98,11 +104,18 @@ protected RestChannelConsumer prepareRequest(final RestRequest request, final No
                     }
 
                     performanceAnalyzerController.updateLoggingState(shouldEnable);
+                } else if (request.path().contains(BATCH_METRICS_CONFIG_PATH)) {
+                    if (shouldEnable && !performanceAnalyzerController.isPerformanceAnalyzerEnabled()) {
+                        return getChannelConsumerWithError("Error: PA not enabled. Enable PA before turning Batch Metrics on");
```

Comment:
I don't think this will help our users. Our users will already know about PA. Also, I just don't find this as a necessity -- I think it would just be unnecessary help information.